fix: add default values to SBOM action

patheard opened the pull request:
# Summary
Default the project version and working directory so they don't
need to be specified.

Also fix the `Invalid project type` error message.

# Related
* cds-snc/platform-sre-security-support#94

dinophile reviewed (approved):
:shipit:

mohamed-cds commented on terragrunt/aws/cloud_asset_inventory/vpc.tf:
```diff
@@ -2,9 +2,9 @@ module "vpc" {
-  cidr            = "10.0.8.0/22"    # Reserve 1,022 IP addresses for VPC. 10.0.9.0/24 and 10.0.10.0/24 are flexible.
-  public_subnets  = ["10.0.8.0/24"]  # Reserve 254 IP addresses for public subnets
-  private_subnets = ["10.0.11.0/24"] # Reserve 254 IP addresses for private subnets
+  cidr            = var.cloud_asset_inventory_cidr # Reserve 1,022 IP addresses for VPC. 10.0.9.0/24 and 10.0.10.0/24 are flexible.
```
added in both places so we maintain context

patheard committed:
fix: add default values to SBOM action

Default the project version and working directory so they don't
need to be specified.

Also fix the `Invalid project type` error message.
